src/python/k4a/docs/building.md

```diff
@@ -79,14 +79,14 @@ The following tools are optional:
    folder named libk4a\<ver\>, where \<ver\> is the version major.minor of the k4a library
    (e.g. _libs constains libk4a1.4/libdepthengine.so.2.0).
```

Comment:
This comment ("The depth engine library must be in a folder named libk4a\<ver> ... ") is no longer relevant due to the changes, right?

Reply:
Correct. Removed it. Thanks for catching.